plugins/obs-webrtc/whip-output.cpp

```diff
@@ -347,28 +349,58 @@ bool WHIPOutput::Connect()
 		return false;
 	}
 
-	if (location_header.empty()) {
+	long redirect_count = 0;
+	curl_easy_getinfo(c, CURLINFO_REDIRECT_COUNT, &redirect_count);
+
+	if (location_headers.size() < static_cast<size_t>(redirect_count) + 1) {
 		do_log(LOG_WARNING,
```

Comment:
This should probably be `LOG_ERROR` now.

Reply:
Changed to `LOG_ERROR`